homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -13,6 +13,8 @@
     FLOW_OFF_ICON,
     FLOW_ON_ICON,
     IDLE_ICON,
+    NO_NOTIFICATION_ICON,
+    NOTIFICATION_ICON,
 )
 from .sensor import SmileSensor
```

Comment:
The binary sensor platform should not depend on the sensor platform.

Reply:
Will rewrite